packages/xstate-wallet/src/store/channel-store-entry.ts

```diff
@@ -257,6 +257,11 @@ export class ChannelStoreEntry {
     const {signedStates} = this;
     const turnNums = _.map(signedStates, s => parseInt(s.turnNum.toHexString(), 16));
 
+    const duplicateTurnNums = turnNums.some((t, i) => turnNums.indexOf(t) != i);
+    if (duplicateTurnNums) {
+      logger.error({signedStates: _.map(signedStates, s => s.turnNum.toHexString())});
```

Comment:
```suggestion
      logger.error(
        {signedStates: _.map(signedStates, s => s.turnNum.toHexString())},
        Errors.duplicateTurnNums
      );
```

Nice to have a string to filter by in the browser